ow_lander/urdf/lander.xacro

```diff
@@ -543,15 +543,6 @@
       </plugin>
     </visual>
     <selfCollide>1</selfCollide>
-    <!-- Turn off collisions with the terrain so scoop can go underground. For
-         this to work, the terrain must use a different collide_bitmask. -->
-    <collision>
-      <surface>

```

Comment:
Collisions are currently disabled for the hand as well. We should probably also remove that code block, unless there is some reason to keep it that I don't know about.

Reply:
I did try this before actually submitting this PR and it did actually cause slowness to happen

Reply:
Weird. If we're going to keep it, it would be a good idea to amend that comment to explain that is is a performance booster. Later on we should think about disabling collisions between other arm parts and the terrain; maybe we can get even more performance boost.

Reply:
addressed